resources/v2/03_students_subjects.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE students_subjects
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    student_id integer NOT NULL,
+    subject_id integer NOT NULL,
+    created_at timestamp
+    WITH time zone DEFAULT NOW(),
+    updated_at timestamp
+    WITH time zone,
+    deleted_at timestamp
+    WITH time zone,
+    CONSTRAINT students_subjects_student_id FOREIGN KEY (student_id) REFERENCES students(id),
+    CONSTRAINT students_subjects_subject_id FOREIGN KEY (subject_id) REFERENCES subjects(id)
+);
+
+CREATE INDEX student_id ON students_subjects USING btree (student_id)
+CREATE INDEX subject_id ON students_subjects USING btree (subject_id)
```

Comment:
THe index name, try to keep it a little different from the field you are trying to index. It would be a little better

Reply:
Took the reference from the template. Also, can you suggest a better naming for this ? I can't think for a better alternative name for the same

Reply:
idx_student_subjects_student_id
idx_student_subjects_subject_id

So idx_table_name_column_name